src/cli/update.ts

```diff
@@ -55,7 +55,7 @@ export default function (program: RootCmd) {
 
         console.log(`✅ Module updated successfully`);
       } catch (e) {
-        console.error(e.message);
+        console.error(`Error updating Pepr module: ${e.message}`);
```

Comment:
```suggestion
        console.error(e, `Error updating Pepr module`);
```

Reply:
All messages updated to display complete `e`